packages/compartment-mapper/package.json

```diff
@@ -14,7 +14,9 @@
   "scripts": {
     "build": "rollup --config rollup.config.js",
     "clean": "rm -rf dist",
-    "lint": "eslint '**/*.js'",
+    "lint": "yarn lint:types && yarn lint-check",
+    "lint:types": "tsc -p jsconfig.json",
+    "lint-check": "eslint '**/*.js'",
```

Comment:
Let’s call this `lint:js`. `link-check` is synonymous with `lint` and I’d like to converge on just `lint`.